### PR TITLE
Add authors and LICENSE file

### PR DESCRIPTION
**Pull request by Zales0123**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,6 +3,16 @@
...
     "description": "Sylius Plugin for order creation in Admin panel",
     "license": "MIT",
+    "authors": [
+        {
+            "name": "Mateusz Zalewski",
+            "homepage": "http://mpzalewski.com.pl"
+        },
+        {
+            "name": "Grzegorz Sadowski",
+            "email": "sadowskigp@gmail.com"
```
Comment (lchrusciel) [inline composer.json]: Shouldn't we use company emails in this plugin?

File: composer.json [hunk 2]
```diff
@@ -3,6 +3,16 @@
...
+        }
```
Comment (lchrusciel) [inline composer.json]: Usually, we are adding a `Sylius Team` as well with generic email https://github.com/Sylius/MVM/blob/master/composer.json#L20-L23

File: composer.json [hunk 3]
```diff
@@ -3,6 +3,16 @@
     "type": "sylius-plugin",
     "description": "Sylius Plugin for order creation in Admin panel",
     "license": "MIT",
+    "authors": [
+        {
+            "name": "Mateusz Zalewski",
+            "homepage": "http://mpzalewski.com.pl"
```
Comment (lchrusciel) [inline composer.json]: Don't you want to add your email also?

File: composer.json (hunk 1 above)
Comment (Zales0123) [reply composer.json]: I think it's up to the author, Grześ, for example, has this email in his Github account, so I understand he doesn't want to mess it up

File: composer.json (hunk 3 above)
Comment (Zales0123) [reply composer.json]: Nope, all info about me is on my very advanced website 😄

**Pull request merged**